datacatalog/snippets/policytagmanager/create_policytag.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policytagmanager
+
+// [START data_catalog_ptm_create_policytag]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	datacatalog "cloud.google.com/go/datacatalog/apiv1beta1"
+	datacatalogpb "google.golang.org/genproto/googleapis/cloud/datacatalog/v1beta1"
+)
+
+// createPolicyTag creates a policy tag resource under a given parent taxonomy.
+//
+// It optionally accepts a parent ID, which can be used to create a hierarchical
+// relationship between tags.
+func createPolicyTag(parent, displayName, parentPolicyTag string, w io.Writer) (string, error) {
```

Comment:
All of our other samples put `w` or `ctx, w` first. Might switching this?

Reply:
done